src/containers/tud/components/ProgressBar.js

```diff
@@ -0,0 +1,146 @@
+// @flow
+
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+import { Colors, Typography } from 'lattice-ui-kit';
+import { DateTime } from 'luxon';
+
+const { PURPLES, NEUTRALS, ORANGE } = Colors;
+
+const Grid = styled.div`
+  align-items: center;
+  display: grid;
+  grid-template-columns: ${(props) => `${props.completedRatio}fr auto`};
+`;
+
+const Wrapper = styled.div`
+  background: white;
+  margin-bottom: 10px;
+  padding: 20px 0;
+  position: sticky;
+  top: 0;
+  z-index: 10;
+`;
+
+const ProgressIndicator = styled.div`
+  border-top-color: ${(props) => props.color};
+  border-top-style: solid;
+  border-top-width: 5px;
+`;
+
+const ProgressLabelWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding-top: 5px;
+
+  :last-of-type {
+    padding-left: 10px;
+  }
+`;
```

Comment:
Why didn't you use the MUI progress component? I haven't looked at it in depth, but it seems like it'll work.

```
<LinearProgress variant="determinate" value={progress} />
```

https://codesandbox.io/s/0gmrv
https://codesandbox.io/s/b05fw